scipy/stats/stats.py

```diff
@@ -2539,39 +2539,10 @@ def zscore(a, axis=0, ddof=0, nan_policy='propagate'):
     array([[-1.13490897, -0.37830299,         nan, -0.08718406,  1.60039602],
            [-0.91611681, -0.89090508,  1.4983032 ,  0.88731639, -0.5785977 ]])
     """
-    a = np.asanyarray(a)
```

Comment:
I can confirm that this code was moved with few modifications into `zmap`. Changes noted below.